scripts/datenlord-monitor-deploy.sh

```diff
@@ -4,29 +4,33 @@ set -o errexit
 set -o nounset
 set -o xtrace
 
```

Comment:
Better to force users to provide one parameter. When user provide 0 parameter, print a usage.

Reply:
for example
datenlord-monitor-deploy.sh [deploy|undeploy|helm]
deploy: xxx
undeploy: xxx
helm: xxxx

Reply:
Done.